fr.free.Qccrypt.yml

```diff
@@ -0,0 +1,34 @@
+app-id: fr.free.Qccrypt
```

Comment:
because of the way `free.fr` works, a proper app id should be `fr.free.qccrypt.Qccrypt`. This is not dissimilar to code hosting. Upstream controls `gccrypt.free.fr` but not `free.fr`

Reply:
Ok, I will change the name to fr.free.qccrypt.Qccrypt.